tools/c7n_azure/tests/cassettes/KeyVaultTest.test_firewall_rules_not_include_cidr.json

```diff
@@ -0,0 +1,389 @@
+                                "type": "Microsoft.KeyVault/vaults",
+                                "location": "westus",
+                                "tags": {
+                                    "CreatorEmail": "anzoloch@microsoft.com"
+                                }
+                            },
+                            {
+                                "id": "/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourceGroups/anzoloch-test-cc/providers/Microsoft.KeyVault/vaults/anzoloch-cc-kv2",
+                                "name": "anzoloch-cc-kv2",
+                                "type": "Microsoft.KeyVault/vaults",
+                                "location": "westus",
+                                "tags": {
+                                    "CreatorEmail": "anzoloch@microsoft.com"
+                                }
+                            },
+                            {
+                                "id": "/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourceGroups/test_keyvault/providers/Microsoft.KeyVault/vaults/cckeyvault1c6ra5qqpkhceo",
+                                "name": "cckeyvault1c6ra5qqpkhceo",
+                                "type": "Microsoft.KeyVault/vaults",
+                                "location": "southcentralus",
+                                "tags": {}
+                            }
+                        ]
+                    }
+                }
+            }
+        },
+        {
+            "request": {
+                "method": "GET",
+                "uri": "https://management.azure.com/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourceGroups/anzoloch-test-cc/providers/Microsoft.KeyVault/vaults/anzoloch-cc-kv1?api-version=2018-02-14",
+                "body": null,
+                "headers": {}
+            },
+            "response": {
+                "status": {
+                    "code": 200,
+                    "message": "OK"
+                },
+                "headers": {
+                    "content-type": [
+                        "application/json; charset=utf-8"
+                    ],
+                    "date": [
+                        "Fri, 14 Jun 2019 18:48:26 GMT"
+                    ],
+                    "cache-control": [
+                        "no-cache"
+                    ],
+                    "content-length": [
+                        "1249"
+                    ]
+                },
+                "body": {
+                    "data": {
+                        "id": "/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourceGroups/anzoloch-test-cc/providers/Microsoft.KeyVault/vaults/anzoloch-cc-kv1",
+                        "name": "anzoloch-cc-kv1",
+                        "type": "Microsoft.KeyVault/vaults",
+                        "location": "westus",
+                        "tags": {
+                            "CreatorEmail": "anzoloch@microsoft.com"
```

Comment:
Having your email in here a bunch probably just gets you spam :) 

Also these cassettes are much larger than they should be - We have been bad about getting automation in place to keep the recording subscription clean.

I would just go delete all those key vaults so you only have the one provided by the ARM template, then record this again.

Reply:
:) Thanks! Rerecorded cassettes